addons/binding/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/handler/NikoHomeControlThermostatHandler.java

```diff
@@ -180,7 +181,8 @@ private void handleCommandSelection(ChannelUID channelUID, Command command) {
     public void initialize() {
         Configuration config = this.getConfig();
 
-        String thermostatId = (String) config.get(CONFIG_THERMOSTAT_ID);
+        thermostatId = (String) config.get(CONFIG_THERMOSTAT_ID);
```

Comment:
These can also be implemented with a config class and getting it with something like:  `getConfigAs(NikoHomeControlThermostatConfig.class)` (you can leave of the `this`, we only use it if there is naming conflict)

Reply:
Done.